### PR TITLE
Update Quick Start of user guide, reorganise content page

### PR DESCRIPTION
**Pull request by yixiann**
Fixes: https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/131

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -21,250 +28,390 @@ commands, you can have complete control of your perishable goods.
...
 
-> Description: Creates a new item with the provided item_name.
+Thank you for choosing FoodRem! We are delighted to have you as a user and aim to serve you well!
 
----
+To gain the most out from this User Guide, start off with the [Quick Start](#quick-start) section.
+This will give you a brief overview about how to use this application. 
 
-Example:
+It is **highly recommended** reading through the User Guide in a **sequential order** up until the section
```
Comment (Eugenetanwl3881) [inline docs/UserGuide.md:115]: It is highly recommended that you read through (Grammar)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -21,250 +28,390 @@ commands, you can have complete control of your perishable goods.
...
+These commands are invalid:
+- `newn/Potatoesqty/30type/kgbgt/05-09-22` (Removing space between command identifier and flag)
+- `new qty/-48 n/PÖtátÖes bgt/05/09/22 type/|kg|` (Restrictions of placeholders not followed)
 
-#### Tag Create
+Find out more about restrictions in the sections [Flags](#flags), [Placeholders](#placeholders) and 
+[Features](#features).
 
-Command: `tag create TAG_NAME`
+You might also realise that there is a pair of square brackets `[]` surrounding some parameters like `qty/QUANTITY`
```
Comment (Eugenetanwl3881) [inline docs/UserGuide.md:190]: This would sound better:
Notice that there is a pair of square brackets `[]` surrounding some parameters like...

**Review by Eugenetanwl3881 — APPROVED** [review]
Great job in reordering and updating of the UG 👍 
Just some minor grammatical issues that can be fixed before merging (see comments)
LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (yixiann) [reply docs/UserGuide.md]: Note, will make the necessary changes.

**Review by bryanljx — APPROVED** [review]
Looks good!

**Pull request merged**